canonical-web-and-design/cn.ubuntu.com
# new fields and rows for desktop

### Done

Added some new fields on the download page and added a whole new row.
### QA

Add some content into the new fields in /admin (pages -> desktop) and check that they work as expected.
